docs/pages/choose-an-edition/teleport-cloud/faq.mdx

```diff
@@ -136,42 +136,58 @@ The ability to download recordings for offline viewing will be available in a fu
 
-<Admonition type="note" title="Minimum agent version">
-    For major version upgrades (e.g. 12.x to 13.x), your Teleport tenant will NOT be automatically upgraded if any
-    Teleport agents connected to the cluster are not compatible (> 1 major version behind).  This is done to ensure
-    backwards compatibility. To get version information on your agents versions
-    see [here](#how-can-i-find-version-information-on-connected-agents).
+If you have Teleport agents connected to a Teleport Team or Teleport Enterprise cluster that are
+more than one major version behind, you might experience compatibility issues if your cluster is 
+automatically upgraded but agents aren't. 
 
-    Please reach out to support if you wish to proceed regardless of this compatibility check.
-</Admonition>
+To get version information for your agents, see [How can I find version information on 
+connected agents?](#how-can-i-find-version-information-on-connected-agents).
+For information about enrolling in automatic agent updates, see [Enroll an agent into automatic 
+updates](../../management/operations/enroll-agent-into-automatic-updates/).
 
-### Are upgrades times configurable for my Teleport Enterprise Cloud tenant?
+For more information about automatic upgrades and compatibility issues, contact 
+[Teleport support](https://goteleport.com/support/).
+
+### Are upgrade times configurable for Teleport Enterprise Cloud?
 
 Yes, you can set the window start time that works best for your organization
 for upgrades. Teleport provides a scheduled maintenance time window for any
 upgrades, patches and other required updates for all tenants. An upgrade for
 your tenant will not start earlier than the window start time you set.
 
-To set the window start time select your username in the Web UI, then the Help
-& Support option from the drop-down. The Scheduled Upgrades section within that
-page allows you to configure the window start time.
+To set the window start time, select your username in the Teleport Web UI, then 
+click **Help & Support**. Under Scheduled Upgrades, you can see the current
+maintenance window start time. Click **Edit** to set a different start
+time for scheduled maintenance.
 
 Teleport recommends subscribing to the Teleport Enterprise Cloud updates at
 [status.teleport.sh](https://status.teleport.sh) to keep up to date on
 scheduled maintenance.
 
+### When are agents automatically updated?
+
+Teleport Team and Teleport Enterprise must be set to receive automatic updates
+to use the Teleport Cloud version server for automatic agent updates. With automatic 
+agent upgrades, agents periodically check the version server for new releases and 
+update the software when new versions are found.
+
+If you enroll in automatic agent upgrades, Teleport agents are automatically 
+updated from one to a few days after your Teleport cluster is upgraded during 
+a scheduled maintenance period.
```

Comment:
This should really just be "Teleport agents are automatically updated during your scheduled maintenance period."

Reply:
@xinding33 just FYI, the issue implies that "during your scheduled maintenance period" isn't the case:

> Currently after a cloud upgrade the version for the agents can be updated 1 or multiple days after.